frontend/src/metabase/writeback/components/ActionCreator/FormCreator/FormCreator.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { t } from "ttag";
+
+import {
+  FormItemWrapper,
+  FormCreatorWrapper,
+  FormItemName,
+  Input,
+  FormSettings,
+  EmptyFormPlaceholderWrapper,
+} from "./FormCreator.styled";
+
+import Icon from "metabase/components/Icon";
+
+import type { TemplateTag } from "metabase-types/types/Query";
```

Comment:
Let's keep local imports (the styled file here) at the bottom of the imports list. [See the style guide](https://www.metabase.com/docs/latest/developers-guide/frontend.html#style-guide)